ruby/README.md

```diff
@@ -14,11 +14,27 @@ Install the gem using
 Alternately, include "rcloadenv" in your application's Gemfile.
 
 You may then invoke the "rcloadenv" binary. You must pass the name of the
-runtime config resource, and then the command to execute. For example, if
-the gem is present in the bundle for your Rails app, you can execute:
+runtime config resource, and then optionally a command to execute, delimited
+by two dashes `--`.
+
+If a command is provided, rcloadenv loads the configuration into environment
+variables and runs the command. For example, if the gem is present in the
+bundle for your Rails app, you can execute:
 
     bundle exec rcloadenv my-config -- bin/rails s
 
+to run Rails directly with the loaded environment.
+
+If a command is _not_ provided_, rcloadenv loads the configuration and prints
+it to stdout in dotenv-compatible format. For example, you can execute:
+
+    bundle exec rcloadenv my-config > .env
+
+to create a `.env` file suitable for loading with the
+[dotenv](https://github.com/bkeepers/dotenv/) gem.
+
+### Accessing runtime config data
```

Comment:
Could you add instructions for using dotenv with GAE flexible environment?

Reply:
@frankyn I'd like to hold off on that for the moment. (I sent you a separate email with details.)